ravenframework/CodeInterfaceClasses/SIMULATE3/SimulateInterface.py

```diff
@@ -0,0 +1,186 @@
+# Copyright 2017 Battelle Energy Alliance, LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Created on August 01, 2022
+Last modified  on August 25, 2022
+
+@author: khnguy22
+
+comments: Interface for Simulate3 Simulation 
+"""
+from __future__ import division, print_function, unicode_literals, absolute_import
+
+import os
+import copy
+import shutil
+from ravenframework.utils import utils
+import xml.etree.ElementTree as ET
+
+from GenericCodeInterface import GenericParser
+import SpecificParser
+from ravenframework.CodeInterfaceBaseClass import CodeInterfaceBase
+from SimulateData import SimulateData
+
+class Simulate(CodeInterfaceBase):
+  """
+    Simulate Interface. Reading output from simulate then export to csv dat file.
+  """
+  def __init__(self):
+    """
+      Constructor
+      @ In, None
+      @ Out, None
+    """
+    CodeInterfaceBase.__init__(self)
+    self.sequence = []   # this contains the sequence that needs to be run. [simulate]
+    self.outputRoot = {} # the root of the output sequences
+
+  def _readMoreXML(self,xmlNode):
+    """
+      Function to read the portion of the xml input that belongs to this specialized class and initialize
+      some members based on inputs. This can be overloaded in specialize code interface in order to
+      read specific flags
+      @ In, xmlNode, xml.etree.ElementTree.Element, Xml element node
+      @ Out, None.
+    """
+    CodeInterfaceBase._readMoreXML(self,xmlNode)
+    sequence = xmlNode.find("sequence")
+    if sequence is None:
+      self.sequence = ['simulate'] #may be no need 
+    else:
+      self.sequence = [elm.strip() for elm in sequence.text.split(",")]
+
```

Comment:
@Nhatkhang and @wangcj05, does this give the user the option if he did not provide a sequence to default to simulate? Do we really want that?